frontend/__tests__/hooks/useVerifyBoardEffect.test.tsx

```diff
@@ -1,44 +1,150 @@
-import { ERROR_MESSAGE_TIME_DURATION, MOCK_BOARD_VERIFY_REQUEST_PARAMS, VERIFY_FAILED } from '../fixtures';
-import { InternalServerException } from '@src/exceptions/InternalServerException';
 import { useVerifyBoardEffect } from '@src/hooks/useVerifyBoardEffect';
-import { boardClient } from '@src/clients/board/BoardClient';
-import { act, renderHook } from '@testing-library/react';
+import { act, renderHook, waitFor } from '@testing-library/react';
+import { MOCK_BOARD_URL_FOR_JIRA } from '@test/fixtures';
+import { setupServer } from 'msw/node';
 import { HttpStatusCode } from 'axios';
 
+import { Iron } from '@mui/icons-material';
+import { rest } from 'msw';
+
+const mockDispatch = jest.fn();
+jest.mock('react-redux', () => ({
+  ...jest.requireActual('react-redux'),
+  useDispatch: () => mockDispatch,
+}));
+
+jest.mock('@src/hooks/useAppDispatch', () => ({
+  useAppSelector: () => ({ type: 'Jira' }),
+}));
+
+const server = setupServer();
+
 describe('use verify board state', () => {
-  it('should initial data state when render hook', async () => {
+  beforeAll(() => server.listen());
+  afterAll(() => {
+    jest.clearAllMocks();
+    server.close();
+  });
+  it('when hook render given none input  then should got initial data state ', async () => {
```

Comment:
update your test description with our UT convension `should .... when ...given`

Reply:
fixed